users/test_runner.py

```diff
@@ -0,0 +1,44 @@
+"""Unit test runner for users service.
+
+Test runner that runs several different test modules.
+To see the tests, look at the sibling test_*.py files."""
+
+# Authors: mukobi
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import sys
+
+import test_authorization
+import test_user_upsertion
+
+
+def generate_suite():
+    """Generates and loads a test suite."""
+    loader = unittest.TestLoader()
```

Comment:
Consider using [`loader.discover(.)`](https://docs.python.org/3/library/unittest.html#unittest.TestLoader.discover) rather than explicitly loading the modules.

`os.path.dirname(__file__)` may be helpful for finding the current file's directory.

Reply:
Thanks, makes things much cleaner. Done in https://github.com/knative-portability/large-events/pull/10/commits/8f14a6ce43a7e631cb9a26e0e97c80cc15d4174b